### ISSUE TITLE
Typo in torch.optim.lr_scheduler.LinearLR example

### ISSUE THREAD

**Issue by aaronzs** [pytorch/pytorch#71544]
### 📚 The doc issue

Line 6 in the example of the LinearLR scheduler, the `lr` should be `0.05` rather than `0.005`.

https://pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.LinearLR.html#torch.optim.lr_scheduler.LinearLR

```
# Assuming optimizer uses lr = 0.05 for all groups
# lr = 0.025    if epoch == 0
# lr = 0.03125  if epoch == 1
# lr = 0.0375   if epoch == 2
# lr = 0.04375  if epoch == 3
# lr = 0.005    if epoch >= 4  <- HERE
scheduler = LinearLR(self.opt, start_factor=0.5, total_iters=4)
for epoch in range(100):
    train(...)
    validate(...)
    scheduler.step()
```

### Suggest a potential alternative/fix

`lr = 0.005` -> `lr = 0.05`

cc @brianjo @mruberry @vincentqb @jbschlosser @albanD

**Comment by albanD** [comment]
Hi!
This is already fixed in the latest doc: https://pytorch.org/docs/master/generated/torch.optim.lr_scheduler.LinearLR.html?highlight=linearlr#torch.optim.lr_scheduler.LinearLR and will be included in the next release.
Thanks for the report!
